Automattic/jetpack
# Plans: update links on single products

Fixes https://github.com/Automattic/jetpack/issues/15183
Fixes #15244

#### Changes proposed in this Pull Request:

* Moves “Which backup option is best for me?” link to the Backup card.
* Replaces regular link in Search for External Link component.
* Adds click tracking to the latter.

#### Before

![image](https://user-images.githubusercontent.com/390760/78170095-57f07600-744a-11ea-9421-3f83697822dc.png)

#### After

<img width="1069" alt="Screen Shot 2020-04-02 at 3 07 51 PM" src="https://user-images.githubusercontent.com/4044428/78299840-c3dae900-74f3-11ea-8e5d-55742c276485.png">

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* No.

#### Testing instructions:

* Fire up this PR.
* Go to `/wp-admin/admin.php?page=jetpack#/plans`
* Ensure:
  * All single product cards look good.
  * Both have links that redirect to their respective LPs.
  * Tracking works.

#### Proposed changelog entry for your changes:

* None.
